qa/rolling-upgrade/build.gradle

```diff
@@ -0,0 +1,135 @@
+/*
+ *  Copyright OpenSearch Contributors
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+
+import org.opensearch.gradle.testclusters.StandaloneRestIntegTestTask
+import org.apache.tools.ant.taskdefs.condition.Os
+
+import java.util.concurrent.Callable
+
+apply from : "$rootDir/qa/build.gradle"
+
+String default_bwc_version = System.getProperty("bwc.version")
+String neural_search_bwc_version = System.getProperty("tests.bwc.version", default_bwc_version)
+boolean isSnapshot = neural_search_bwc_version.contains("-SNAPSHOT")
+String knn_bwc_version_no_qualifier = isSnapshot ? neural_search_bwc_version - "-SNAPSHOT" : neural_search_bwc_version
+String baseName = "neuralSearchBwcCluster-rolling"
+
+List<Provider<RegularFile>> plugins = [provider(new Callable<RegularFile>(){
+    @Override
+    RegularFile call() throws Exception {
+        return new RegularFile() {
+            @Override
+            File getAsFile() {
+                return configurations.zipArchive.asFileTree.matching{include "**/opensearch-ml-plugin-${opensearch_build}.zip"}.getSingleFile()
+            }
+        }
+    }
+}), provider(new Callable<RegularFile>(){
+    @Override
+    RegularFile call() throws Exception {
+        return new RegularFile() {
+            @Override
+            File getAsFile() {
+                return configurations.zipArchive.asFileTree.matching{include "**/opensearch-knn-${opensearch_build}.zip"}.getSingleFile()
+            }
+        }
+    }
+}), rootProject.tasks.bundlePlugin.archiveFile]
+
+// Creates a test cluster of previous version and loads k-NN plugin of bwcVersion
+testClusters {
+    "${baseName}" {
+        testDistribution = "ARCHIVE"
+        versions = [neural_search_bwc_version, opensearch_version]
+        numberOfNodes = 3
+        plugin(project.tasks.zipBwcMlCommonsPlugin.archiveFile)
+        plugin(project.tasks.zipBwcKnnPlugin.archiveFile)
+        plugin(project.tasks.zipBwcPlugin.archiveFile)
+        jvmArgs("-Xms1g", "-Xmx4g")
+        setting 'path.repo', "${buildDir}/cluster/shared/repo/${baseName}"
+        setting 'http.content_type.required', 'true'
+        environment "LD_LIBRARY_PATH", "${buildDir}/testclusters/${baseName}-0/distro/${knn_bwc_version_no_qualifier}-ARCHIVE/plugins/opensearch-knn/knnlib;${buildDir}/testclusters/${baseName}-0/distro/${knn_bwc_version_no_qualifier}-ARCHIVE/plugins/opensearch-knn/lib"
+        if (Os.isFamily(Os.FAMILY_WINDOWS)) {
+            // While running on Windows OS, setting the PATH environment variable to include the paths to dlls of JNI libraries and windows dependencies
+            environment('PATH', System.getenv('PATH') + ";$rootDir/jni/release" + ";$rootDir/src/main/resources/windowsDependencies")
```

Comment:
update the comment and remove jni path like mentioned in above comment

Reply:
done